lib/src/Uncertainty/Bayesian/MetropolisHastingsImplementation.cxx

```diff
@@ -192,31 +181,28 @@ Point MetropolisHastingsImplementation::getRealization() const
       throw InvalidArgumentException(HERE) << "The initial state should have non-zero posterior probability density";
   }
 
-  // for each new sample
-  for (UnsignedInteger i = 0; i < size; ++ i)
+  Point newState(currentState_);
+  const Point candidate(getCandidate());
```

Comment:
Compared to other calibration classes, it may happen that the word "candidate" is inappropriate. Would `startingPoint` clearer in this setting?

Reply:
It is not a starting point. It is the proposal for the next step of the Markov chain. We decided to call the instrumental distribution "proposal distribution", so we settled on "candidate" for the proposed point to avoid confusion.

Reply:
I don't think you can compare this with other calibration classes, because this is a purely MCMC notion, and other calibration classes do not use MCMC.